.gitpod.yml

```diff
@@ -0,0 +1,34 @@
+tasks:
+  - init: npm install && npm run build
+    command: npm run start
+
+vscode:
+  extensions:
```

Comment:
These are already listed in `.vscode/extensions.json` and a couple of the recommended extensions are missing. Does Gitpod recognize them from the `.vscode` directory? Can we omit this?

Reply:
@claviska You are right!
We can omit all the extensions, because it will take it automatically from `.vscode/extensions.json`
https://www.gitpod.io/docs/references/gitpod-yml#vscodeextensions